assets/js/blocks/price-filter/block.js

```diff
@@ -71,6 +84,8 @@ const PriceFilterBlock = ( { attributes, isEditor = false } ) => {
 		''
 	);
 
+	const [ hasSetPriceDefault, setHasSetPriceDefault ] = useState( false );
```

Comment:
Do you think we should add a comment above this piece of state mentioning that it is to track the defaults we set only on the PHP-rendered Classic Block pages? As its not used for the All Products block.